tools/wpt/browser.py

```diff
@@ -1104,6 +1104,38 @@ def version(self, binary=None, webdriver_binary=None):
         return None
 
 
+class WebKitGTKMiniBrowser(WebKit):
+
+    def find_binary(self, venv_path=None, channel=None):
+        libexecpaths = ["/usr/libexec/webkit2gtk-4.0",
+                        "/usr/lib/x86_64-linux-gnu/webkit2gtk-4.0"]
+        # If GCC is available use it to detect this machine triplet,
```

Comment:
This is a bit convoluted, and I expect it's hard to guess what one has to install to run this product, or to work backwards from this to what the Debian package name is.

Can you add a piece of documentation like https://github.com/web-platform-tests/wpt/blob/master/docs/running-tests/chrome.md that says what to packages to install to run this product?

Reply:
Sure